serving/resource_model_serving.go

```diff
@@ -17,16 +17,19 @@ func ResourceModelServing() *schema.Resource {
 		serving.CreateServingEndpoint{},
 		func(m map[string]*schema.Schema) map[string]*schema.Schema {
 			m["name"].ForceNew = true
-			common.MustSchemaPath(m, "config", "served_models", "scale_to_zero_enabled").Required = false
-			common.MustSchemaPath(m, "config", "served_models", "scale_to_zero_enabled").Optional = true
-			common.MustSchemaPath(m, "config", "served_models", "scale_to_zero_enabled").Default = true
-			common.MustSchemaPath(m, "config", "served_models", "name").Computed = true
-
-			common.MustSchemaPath(m, "config", "served_models", "workload_type").Default = "CPU"
-			// TODO: `config.served_models.workload_type` should be a `Optional+Computed` field. Also consider this for other similar fields.
-			// In this scenario, if a workspace does not have GPU serving, specifying `workload_type` = 'CPU' will get empty response from API.
-			common.MustSchemaPath(m, "config", "served_models", "workload_type").DiffSuppressFunc = func(k, old, new string, d *schema.ResourceData) bool {
-				return old == "" && new == "CPU"
+			for _, name := range []string{"served_models", "served_entities"} {
+				common.MustSchemaPath(m, "config", name).Computed = true
+				common.MustSchemaPath(m, "config", name, "name").Computed = true
+				common.MustSchemaPath(m, "config", name, "environment_vars").Computed = true
+				common.MustSchemaPath(m, "config", name, "scale_to_zero_enabled").Required = false
+				common.MustSchemaPath(m, "config", name, "scale_to_zero_enabled").Optional = true
+				common.MustSchemaPath(m, "config", name, "scale_to_zero_enabled").Default = true
+				common.MustSchemaPath(m, "config", name, "workload_type").Default = "CPU"
```

Comment:
the API would fail this, as you are now sending both `served_models` and `served_entities` in the payload